server/data/data-jdbc/pom.xml

```diff
@@ -37,6 +37,12 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>apache-mailet-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>apache-mailet-base</artifactId>
```

Comment:
I don't like this dependency, it's very close to a circular dependency. Couldn't we move this MimeMessageBuilder elsewhere instead ?